internal/overlord/servstate/handlers.go

```diff
@@ -162,11 +169,48 @@ func (m *ServiceManager) doStart(task *state.Task, tomb *tomb.Tomb) error {
 			}
 			releasePlan()
 		}
-		return fmt.Errorf("cannot start service: exited quickly with code %d", cmd.ProcessState.ExitCode())
+
+		logs, err := getLastLogs(logBuffer)
+		if err != nil {
+			logger.Noticef("Cannot read service %q logs: %v", req.Name, err)
+		}
+
+		msg := fmt.Sprintf("service exited too quickly with code %d", cmd.ProcessState.ExitCode())
+		if logs != "" {
+			msg += ":\n" + logs
```

Comment:
Per note on the last review, can we please use OutputErr here?  It is oriented exactly for this use case: merging some output and a related error value. It handles a few different cases, has formatting to isolate the text, and it works well when there is no text as well, so you can just blindly pass the text in without worrying about it. Also, if it happens to not be great for a certain case, we can fix a single location and have it improved everywhere consistently.

Reply:
`OutputErr` is a bit of a strange one, as it *doesn't* show the `err` at all if there is output. But in this case the `err.Error()` message is useful for context, and includes the exit code. For example, with `OutputErr`:

```
cannot perform the following tasks:
- Start service "sleepx":
-----
sleep: invalid time interval ‘x’
Try 'sleep --help' for more information.
-----
```

Rather than the more helpful:

```
cannot perform the following tasks:
- Start service "sleepx": service exited too quickly with code 1:
    sleep: invalid time interval ‘x’
    Try 'sleep --help' for more information.
```

I'm fine with the additional `-----` formatting here if you want, but the lack of "service exited too quickly with code 1" is not an improvement. We could change `OutputErr` to include the err as well, and maybe that'd be an improvement for everyone? But it seems it was intentionally designed that way, though I'm not quite sure why -- error information is lost.

Reply:
This is moot now: `OutputErr` is no longer relevant now that we're putting the service output logs in the task's log rather than returning it in the change error.